site/source/user-manual/service-guides/lightning/connecting-lnbits.rst

```diff
@@ -0,0 +1,63 @@
+.. _connecting-lnbits:
+
+======
+LNbits
+======
+
+The concept
+-----------
+
+LNbits allows you to create a wallet that makes use of your node with only an alloted amount of sats. This restriction can be very helpful for if you only want to have a small amount for spending on your phone without making your entire lightning balance available. You can even allow other people to have wallets you create for them - think giving your children an allowance. They can start with a set amount (can be zero) and simply spend what you initially make available. They can also earn more for themselves as they will have the freedom to issue their own invoices - all while making use of your node.
+
+.. tip:: All wallets created this way are ultimately bound by the capacity of your node. If one wallet is allocated 10,000 sats but your underlying node only has 9000 sats of outbound capacity, payments will simply fail.
+
+What lightning node should I use?
+---------------------------------
+
+This will work for both **Core Lightning (CLN)** and **LND** *but* if you want to connect mobile wallets :ref:`Zeus<zeus-lnbits>` or :ref:`BlueWallet<blue-wallet>` to LNbits then **this will only work with LND** as the underlying node. This is because the LNDhub plugin will be required. 
```

Comment:
```suggestion
This will work for both **Core Lightning (CLN)** and **LND** *but* if you want to connect :ref:`Zeus<zeus-lnbits>` or :ref:`BlueWallet<blue-wallet>` to LNbits then **this will only work with LND** as the underlying node. This is because the LNDhub plugin will be required. 
```